.markdown-link-check.json

```diff
@@ -9,9 +9,6 @@
     },
     {
       "pattern": "^https://github.com/.*/discussions/new"
-    },
-    {
-      "pattern": "^https://www\\.autosar\\.org/*"
```

Comment:
Will wait for Autosar's homepage to be fixed for a while.
https://github.com/autowarefoundation/autoware-documentation/pull/289

Reply:
It's already accessible but the return code is 0.
https://www.autosar.org/fileadmin/standards/adaptive/22-11/AUTOSAR_RS_CPP14Guidelines.pdf

Reply:
Updated the sync setting in https://github.com/autowarefoundation/autoware-documentation/pull/341.